packages/datadog-plugin-jest/src/index.js

```diff
@@ -197,8 +200,10 @@ class JestPlugin extends CiPlugin {
     const extraTags = {
       [JEST_TEST_RUNNER]: runner,
       [TEST_PARAMETERS]: testParameters,
-      [TEST_FRAMEWORK_VERSION]: frameworkVersion,
-      [TEST_SOURCE_START]: testStartLine
+      [TEST_FRAMEWORK_VERSION]: frameworkVersion
+    }
+    if (testStartLine) {
```

Comment:
You are deleting the `testStartLine` parameter here: https://github.com/DataDog/dd-trace-js/pull/3399/files#diff-74a96f6d5ac2137dcdae2e86565193c9f9a46b10276918a309dd19f974da1353L133

With that change, is it possible to receive `testStartLine` property in `test` in this method?

Reply:
yeah, from the test skipped case: https://github.com/DataDog/dd-trace-js/blob/master/packages/datadog-instrumentations/src/jest.js#L161